pyproject.toml

```diff
@@ -8,6 +8,9 @@ version = "0"
 
 [tool.mypy]
 mypy_path = 'src'
+[[tool.mypy.overrides]]
+ module = ["mockssh"]
```

Comment:
Is this extra space relevant?  I don't do toml.

Reply:
No, just a typo. Fixed.